packages/neo4j/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@next-auth/neo4j-adapter",
+  "version": "0.1.0",
+  "description": "neo4j adapter for next-auth.",
+  "homepage": "https://next-auth.js.org",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/next-auth/issues"
+  },
+  "author": "Davey Brown",
+  "main": "dist/index.js",
+  "license": "ISC",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "oauth",
+    "neo4j"
+  ],
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test:watch": "./tests/test.sh -w",
+    "test": "./tests/test.sh",
+    "build": "tsc"
+  },
+  "files": [
+    "README.md",
+    "dist"
+  ],
+  "peerDependencies": {
+    "neo4j-driver": "^4.0.0",
+    "next-auth": ">=4"
```

Comment:
```suggestion
    "next-auth": ">4 || 4.0.0-beta.1 - 4.0.0-beta.x",
```

As per #274

Reply:
@EarthlingDavey if you fix this, I think we can merge it as is, but we will probably do some cleanup on the code if that is fine by you! Thank you 🙏

Reply:
@balazsorban44 sure, I'll do this over the weekend 👍 And of course, I would appreciate the cleanup. If you tag me I'll read through the changes to learn from them. Cheers, ha det - have a good weekend

Reply:
Never mind,  the suggested change is really small, and I can just merge in as-is and get to work on the cleanup stuff. I would like to get everything ready for the v4 release in time, which is in less than 2 weeks. :slightly_smiling_face:

Reply:
This is awesome. Thanks man!